UltraStar Play/Assets/Scenes/Sing/SingSceneControl.cs

```diff
@@ -98,9 +99,23 @@ public static SingSceneControl Instance
     [Inject(UxmlName = R.UxmlNames.inputLegend)]
     private VisualElement inputLegend;
 
+    [Inject(UxmlName = R.UxmlNames.webcamRenderContainer)]
```

Comment:
Please create a new SingSceneWebcamControl for all this. I think it makes sense to create a new GameObject in the scene for this.

Reply:
Unfortunately I do not understand excatly what you mean... Can you specify it a bit more for me?